htmx-spring-boot/src/test/java/io/github/wimdeblauwe/htmx/spring/boot/mvc/HtmxHandlerMethodArgumentResolverTest.java

```diff
@@ -4,22 +4,29 @@
 import org.mockito.ArgumentCaptor;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
-import org.springframework.boot.test.mock.mockito.MockBean;
 import org.springframework.security.test.context.support.WithMockUser;
+import org.springframework.stereotype.Controller;
+import org.springframework.stereotype.Service;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.bean.override.mockito.MockitoBean;
 import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
 
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.mockito.Mockito.verify;
 import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.view;
 
 
-@WebMvcTest(HtmxHandlerMethodArgumentResolverTestController.class)
+@WebMvcTest(HtmxHandlerMethodArgumentResolverTest.TestController.class)
+@ContextConfiguration(classes = HtmxHandlerMethodArgumentResolverTest.TestController.class)
 @WithMockUser
 class HtmxHandlerMethodArgumentResolverTest {
     @Autowired
     private MockMvc mockMvc;
-    @MockBean
+    @MockitoBean
```

Comment:
What is the difference between `@MockBean` and `@MockitoBean` ?

Reply:
[MockBean](https://docs.spring.io/spring-boot/api/java/org/springframework/boot/test/mock/mockito/MockBean.html) is deprecated, for removal: This API element is subject to removal in a future version.
since 3.4.0 for removal in 3.6.0 in favor of [MockitoBean](https://docs.spring.io/spring-framework/docs/6.2.x/javadoc-api/org/springframework/test/context/bean/override/mockito/MockitoBean.html)

Reply:
Oh, I did not know that.